extensions-core/protobuf-extensions/src/test/java/org/apache/druid/data/input/protobuf/ProtobufInputRowParserTest.java

```diff
@@ -76,6 +77,19 @@ public void setUp()
         null
     );
 
+    flattenParseSpec = new JSONParseSpec(
```

Comment:
nit: should this be named `flatParseSpec` since it is for flat data and _will not_ flatten the data since it has a `null` flattenSpec?

Reply:
yeah you're right.